### DISCUSSION TITLE
Add Sepolia Network to brownie network list

### DISCUSSION THREAD

**Discussion by purplestoke** [smartcontractkit/full-blockchain-solidity-course-py#1850] [General]
Took me a while to figure out how to add the Sepolia network to the brownie networks list so I figured I'd put it here for anyone else. 

Run this command in the terminal be sure to use your Infura project id  

brownie networks add Ethereum sepolia host="https://sepolia.infura.io/v3/YOUR_PROJ_ID" chainid=11155111

**Comment by Hrih** [comment]
Saved me tons of work, thanks! 

By the way, when I ran the command line above, getting into the brownie publish_source part , I got this "Explorer API not set for this network error". According to eth-brownie doc, I was supposed to add a field called **explorer** since sepolia is a live network. 

The explorer api of sepolia is elusive and I wonder how I can fix my network settings at this stage.

**Comment by cromewar** [comment]
Hello @purplestoke thanks for the contribution on this!
@Hrih to add the explorer you can run again this command: 

`brownie networks add Ethereum sepolia host="https://sepolia.infura.io/v3/YOUR_PROJ_ID" chainid=11155111 explorer=https://sepolia.etherscan.io/`

As you already have the network you can delete it first:

`brownie networks delete <id>` 

And then add it again.

Cheers.

**Reply by walterthesmart** [reply]
Hi, I am also at the publish_source=True part. Since rinkeby and goerli are kinda deprecated, I have been trying to know if one can publish their code on Sepolia testnet. I keep getting this error
![image](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/assets/47696744/618c70ba-25f4-44ed-bb50-a7704989ea1b)

**Comment by Dahe-Pan** [comment]
Thaks so much, this is really helpful.

**Comment by fabian416** [comment]
Thank you very much, you saved me a lot of time, I appreciate it
